Qadriyah/sm-frontend
# #164495134 Setup React Router

### What does this PR do?

Sets up react router 4 and all integrations

### Description of Task to be completed?

- React router setup
- TravicCI integration
- Codeclimate setup
- Eslint setup
- Hound setup
- Update README.md with the budges

### How should this be manually tested?

- Run npm install to install dependencies.
- Run npm start to run the application.
- On github, navigate to the develop branch to see the budges

### Any background context you want to provide?

### What are the relevant pivotal tracker stories?

[#164495134 #164495144 #164495157](https://www.pivotaltracker.com/n/projects/2317706/stories/164495134)
